package.json

```diff
@@ -4,7 +4,8 @@
   "description": "Seneca RabbitMQ transport",
   "main": "rabbitmq-transport.js",
   "scripts": {
-    "test": "node_modules/.bin/mocha test/*.test.js"
+    "test": "./node_modules/.bin/lab test/*test.js  -r console -v",
+    "setup-test": "./setup.sh"
```

Comment:
isn't testing anything just setting up dock container, are you sure that you want to change it?